internal/clients/clients.go

```diff
@@ -126,10 +126,20 @@ func (c *client) do(req *http.Request, out interface{}) (*http.Response, error)
 	defer resp.Body.Close()
 
 	r := io.LimitReader(resp.Body, c.maxBodySize)
+
+	ct := resp.Header.Get("Content-Type")
+	if ct != "application/json" {
```

Comment:
We need to check the prefix here, not exact value. `Content-Type: application/json; charset=uft8` is a valid header, for example.

Reply:
done